src/apps/routes/schemas/analysis/listing.clj

```diff
@@ -81,7 +84,10 @@
     (describe Boolean "Indicates whether the app is currently disabled.")
 
     :can_share
-    (describe Boolean "Indicates whether or not the analysis can be shared.")))
+    (describe Boolean "Indicates whether or not the analysis can be shared.")
+
+    (optional-key :interactive_urls)
```

Comment:
It looks like this `:interactive_urls` param could go into `BaseAnalysis`, rather than duplicated in `AdminAnalysis` and `Analysis`?